applications/sim_showers_for_trigger_rates.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application simulates showers to be used in trigger rate calculations.
+    Arrays with one (1MST) or four telescopes (4LST) can be used, in case of \
+    mono or stereo trigger configurations, respectively. 
+
+    Simulations are managed by the shower_simulator module.
+    Each run is simulated in a job. Each job is submitted by using the submission \
+    command from the global config settings (see config_template.yml). \
+    The config entry extraCommands can be used to extra commands to be ran in each job,
+    before the actual simulation.
+
+    At the moment, the shower simulations are performed by CORSIKA, which requires \
+    the zstd package. Please, make sure that the command to set your zstd path is \
+    properly set by the extraCommands in config.yml.
+
+    Command line arguments
+    ----------------------
+    array (str, required)
+        Name of the array (1MST, 4LST ...).
+    site (str, required)
+        South or North.
+    primary (str, required)
+        Name of the primary particle (proton, helium ...).
+    nruns (int, optional)
+        Number of runs to be simulated (default=100).
+    nevents (int, optional)
```

Comment:
For this particular application, why does the user have to bother with runs and number of events per run? Would it make sense to introduce e.g., as goal an expected error on the derived trigger rates? I really wouldn't know what to add here as values for nruns and nevents - but I would know that I want to know the trigger rates on an accuracy of XX%

Reply:
My idea was to have default values that are reasonable and expected to be used in regular cases, but still leave the option to change it in case the user knows what he/she is doing.

I think this is fine. We could maybe write that in the description of the application to tell users that the default values are optimal choices for regular cases. 

The current default values are placeholders. I still need to perform this end-to-end simulations to figure out what the reasonable numbers are.

Reply:
I've moved this into issue #123